demo/constants/ContractsABIs.js

```diff
@@ -0,0 +1,7 @@
+import { abi as estateABI } from "../abi/EstateRegistry.json";
```

Comment:
I forget, did you have thoughts on getting the ABIs from over in the SDK in the future?

Reply:
I suppose maybe that's not the best demo of how an external user would use it, though.

Reply:
Mentioning this tool in case it's useful to think about, since it's kind of on this topic https://github.com/JoinColony/trufflepig

Reply:
> I forget, did you have thoughts on getting the ABIs from over in the SDK in the future?

Yes, maybe we can expose ABIs when Decentraland package will be available;

> I suppose maybe that's not the best demo of how an external user would use it, though.

Using the Contract class with their own ABI isn't the user will do something like that?
Maybe expose some ABIs (20,721,etc)  as constants?

> Mentioning this tool in case it's useful to think about, since it's kind of on this topic
https://github.com/JoinColony/trufflepig

Maybe we have to think on both scenarios: 
1) When user under develop process using the SDK (local blockchain / abi from `truffle`/`trufflepig`)
2) When SDK is using to interact with a pre-existing mainnet (without ganache-cli / not deployed by user)

Brainstorm Note: When I was writing item 2, I've thought that we can even suggest to our users development stages when interact with 3rd-party contracts. As we are doing with Decentraland:
- Local deployment from same code and test as contract owner
- Local network fork from testnet (We will do that and test suite should be changed because we'll no longer been owner of contracts);
- Testnet network (Should be a smooth move from last step)
- Local network fork from mainnet;
- Mainnet network.
Assuming that the 3rd-party keeps both testnet/mainnet as Decentraland does.

ABI Note: What do you think about `ethers.js` human readable ABI? Should we allows users instante contracts using that?

Reply:
On my mobile device, but my initial reaction is that that all makes sense and sounds good! Should we extract this discussion to an issue?

Reply:
https://github.com/tasitlabs/tasit/issues/82

Reply:
Thanks! Resolving